WebApplication/Backend/DTO/TimeIntervalDTO.cs

```diff
@@ -0,0 +1,79 @@
+﻿using HealthClinicBackend.Backend.Model.Util;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WebApplication.Backend.DTO
+{
+    public class TimeIntervalDTO
+    {
+        private DateTime start;
+        private DateTime end;
```

Comment:
also we don't need private fields, just public properties (see how other model classes look)